src/adaptation/Config.cc

```diff
@@ -205,7 +205,7 @@ Adaptation::Config::finalize()
         }
     }
 
-    debugs(93,3, HERE << "Created " << created << " adaptation services");
+    debugs(93,3, "Created " << created << " adaptation services");
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.